packages/typespec-go/test/cadlranch/azure/core/lro/lrorpcgroup/custom_client.go

```diff
@@ -0,0 +1,40 @@
+//go:build go1.18
+// +build go1.18
```

Comment:
Do we need to keep generating this?

Reply:
This was copy/paste that predated its removal. Fixed.